app/pandas.ipynb

```diff
@@ -413,9 +532,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# use % operator here"
-   ],
+   "source": "modulus = s1 % s2",
```

Comment:
The modulus operation is defined but not executed. You should print the result to verify the operation.